### PR TITLE
Add Browse Page Filter Translations

### PR DESCRIPTION
**Pull request by chase-manning**
closes: #135

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -79,5 +79,300 @@
...
+    "HeadAccessory": {
+      "name": "Head Accessory",
+      "values": {
+        "mask": "Mask",
+        "hairclip": "Hair Clip",
+        "hairbow": "Hair Bow",
+        "covidmask": "Covid Mask",
+        "catgirl": "Cat Girl",
+        "maidhat": "Maid Hat",
+        "dogemask": "Doge Mask",
```
Comment (0xKiwi) [inline public/locales/en/translation.json:94]: This is really a hat isn't it?

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -79,5 +79,300 @@
...
+        "ears": "Ears",
```
Comment (0xKiwi) [inline public/locales/en/translation.json:95]: Ears? Just ears?

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -79,5 +79,300 @@
...
+        "swimsuit": "Swimsuit",
+        "catgirl": "Cat Girl",
+        "pjs": "PJs",
+        "kimono": "Kimono",
+        "vampire": "Vampire",
+        "maid": "Maid",
+        "wacdonalds": "Wacdonalds",
+        "bunny": "Bunny",
+        "overalls": "Overalls",
+        "plugsuit": "Playsuit"
```
Comment (0xKiwi) [inline public/locales/en/translation.json:138]:
```suggestion
        "plugsuit": "Plugsuit"
```

File: public/locales/en/translation.json [hunk 4]
```diff
@@ -79,5 +79,300 @@
...
+        "surprised": "Surprised",
+        "cat": "Cat",
+        "sad": "Sad",
+        "doki": "Doki",
+        "neutral": "Neutral",
+        "angry": "Angry",
+        "pouty": "Pouty",
+        "araara": "Ara Ara",
+        "smug": "Smug",
+        "ahegao": "Ahego",
```
Comment (0xKiwi) [inline public/locales/en/translation.json:220]:
```suggestion
        "ahegao": "Ahegao",
```

File: public/locales/en/translation.json (hunk 1 above)
Comment (chase-manning) [reply public/locales/en/translation.json:94]: Changed to `Doge Hat` 😄

File: public/locales/en/translation.json (hunk 3 above)
Comment (chase-manning) [reply public/locales/en/translation.json:138]: Fixed, ty 😄

File: public/locales/en/translation.json (hunk 4 above)
Comment (chase-manning) [reply public/locales/en/translation.json:220]: Fixed, 👍🏻

File: public/locales/en/translation.json (hunk 2 above)
Comment (chase-manning) [reply public/locales/en/translation.json]: Added 🐰

File: public/locales/en/translation.json [hunk 5]
```diff
@@ -79,5 +79,300 @@
...
+    },
+    "SpeechBubble": {
+      "name": "Speech Bubble",
+      "values": {
+        "baka": "Baka",
+        "senpai": "Senpai",
+        "oniisama": "Oniisama",
+        "daisuki": "Daisuki",
+        "anonkun": "Anonkun",
+        "pogchamp": "Pogchamp"
```
Comment (Phineas) [inline public/locales/en/translation.json:272]: Should be PogChamp

File: public/locales/en/translation.json [hunk 6]
```diff
@@ -79,5 +79,300 @@
...
+  "traits": {
+    "all": "All",
+    "HeadAccessory": {
+      "name": "Head Accessory",
+      "values": {
+        "mask": "Mask",
+        "hairclip": "Hair Clip",
+        "hairbow": "Hair Bow",
+        "covidmask": "Covid Mask",
+        "catgirl": "Cat Girl",
```
Comment (Phineas) [inline public/locales/en/translation.json:92]: I feel like Catgirl should be one word throughout, thoughts @0xKiwi?

**Review by Phineas — CHANGES_REQUESTED** [review]

File: public/locales/en/translation.json (hunk 6 above)
Comment (0xKiwi) [reply public/locales/en/translation.json:92]: Yeah sounds good

**Pull request merged**